code/part-two/client/source/services/addressing.js

```diff
@@ -79,22 +146,9 @@ export const getOfferAddress = (ownerKey = null, moji = null) => {
       return addressOrDna;
     }
 
-    return getMojiAddress(publicKey, addressOrDna);
+    return getMojiAddress(ownerKey, addressOrDna);
   });
 
   return collectionPrefix + hash(addresses.join(''), 54);
```

Comment:
Should we sort addresses before we join them?

Reply:
Yes. Damn. I'll add a commit.